tests/unit/local/apigw/test_service_error_responses.py

```diff
@@ -16,7 +16,7 @@ def test_lambda_failure_response(self, jsonify_patch, make_response_patch):
         self.assertEqual(response, {"Some Response"})
 
         jsonify_patch.assert_called_with({"message": "Internal server error"})
-        make_response_patch.assert_called_with({"json": "Response"}, 502)
+        make_response_patch.assert_called_with({"json": "Response"}, 500)
```

Comment:
I don't think this should change. Looking at this test, it seems like the Lambda function raises an error vs the issues comments about a logical issue in the code.

It would be helpful to write an integration test for the case detailed in the Github Issue that proves what you intend happens and does not have side affects to other areas of the codebase.

Reply:
We have some integration tests already testing the functionality that local start api returns an error for invoking, as for the integration tests comparing local and aws, that is something we should look at in the future, because there is a difference between the error codes used there so once we fix that we can create an integ test for it.